operator_ui/src/pages/Jobs/New.tsx

```diff
@@ -274,6 +290,27 @@ export const New = ({
             </CardContent>
           </Card>
         </Grid>
+        <Grid item xs={12} lg={4}>
+          <Card style={{ overflow: 'visible' }} className={classes.card}>
+            <CardHeader title="Task list preview" />
+            <Divider />
+            {tasks.format === JobSpecFormats.JSON && tasks.list && (
+              <TaskList tasks={tasks.list as TaskSpec[]} />
+            )}
+            {tasks.format === JobSpecFormats.TOML && tasks.list && (
+              <TaskListDag stratify={tasks.list as Stratify[]} />
+            )}
+            {!tasks.list && (
```

Comment:
this should be a ternary, otherwise it might print `false`?

Reply:
@ainsleybc No this should never print `false`. Updated code with additional test for empty Task list.